documentation/src/bootstrap_node.md

```diff
@@ -0,0 +1,93 @@
+# Bootstrap node
+
+## Introduction
+
+The bootstrap node is the first node a new node contacts when it joins the
+network. It is responsible for providing the new node with a list of other nodes
+in the network, which the new node can then contact to join the network. Every
+Forest node has a list of bootstrap nodes that it can contact to join the
+network. This list is hardcoded into the node but can be modified by the user
+via the configuration file.
+
+## Forest as a bootstrap node
+
+Every Forest node can act as a bootstrap node. That said, running a `stateless`
+node as a bootstrap node is recommended to lower the hardware requirements. The
+`stateless` node does not store the network's state or participate in the
+consensus process. It only serves as a gateway for new nodes to join the
+network.
+
+Stateless node characteristics:
+
+- it connects to the P2P swarm but does not store the state of the network,
+- it does not sync the chain,
+- it does not validate the chain,
+- `Hello` requests' heaviest tipset is the genesis tipset (unless the node was
```

Comment:
```suggestion
- `Hello` requests' heaviest tipset is set to the genesis tipset (unless the node was
```

Reply:
I don't think the additional wording adds any more value to the sentence.

Reply:
I had to read this at least two times to understand what it says. That is more verbose, but clearer, at least to me. Not a blocker.

Reply:
It's your choice whether to merge it is as or incorporate this.